coralproject/talk
# Docker Updates

## What does this PR do?

Introduces the concept of build variables which are used for the Onbuild docker images.

## How do I test this PR?

Run:

1. `docker build -t coralproject/talk:latest -f Dockerfile .`
2. `docker build -t coralproject/talk:latest-onbuild -f Dockerfile.onbuild .` 
3. Create a file `Dockerfile.my_release` with contents:

```
FROM coralproject/talk:latest-onbuild
```

4. `docker build -t coralproject/talk:my_release --build-arg TALK_THREADING_LEVEL=1 -f Dockerfile.my_release .`

Use docker compose with the image tag: `coralproject/talk:my_release` and notice the maximum threading level supported is 1.
